charts/OpenSDS Installation using Helm.md

```diff
@@ -0,0 +1,316 @@
+```
+sudo pvdisplay # Check if physical volume existed
+sudo vgdisplay # Check if volume group existed
+
+mkdir -p /etc/opensds/driver && sudo cat > /etc/opensds/driver/lvm.yaml <<OPENSDS_DRIVER_CONFIG_DOC
+tgtBindIp: 127.0.0.1
+tgtConfDir: /etc/tgt/conf.d
+pool:
+  {{ volume_group_name }}:
+    storageType: block
+    availabilityZone: default
+    extras:
+      dataStorage:
+        provisioningPolicy: Thin
+        isSpaceEfficient: false
+      ioConnectivity:
+        accessProtocol: iscsi
+        maxIOPS: 7000000
+        maxBWS: 600
+      advanced:
+        diskType: SSD
+        latency: 5ms
+OPENSDS_DRIVER_CONFIG_DOC
+```
+
+If you choose `ceph` as backend, you need to configure ceph driver.
+```
+mkdir -p /etc/opensds/driver && sudo cat > /etc/opensds/driver/ceph.yaml <<OPENSDS_DRIVER_CONFIG_DOC
+configFile: /etc/ceph/ceph.conf
+pool:
+  rbd: # change pool name same to ceph pool, but don't change it if you choose lvm backend
+    storageType: block
+    availabilityZone: default
+    extras:
+      dataStorage:
+        provisioningPolicy: Thin
+        isSpaceEfficient: true
+      ioConnectivity:
+        accessProtocol: rbd
+        maxIOPS: 6000000
+        maxBWS: 500
+      advanced:
+        diskType: SSD
+        latency: 5ms
+OPENSDS_DRIVER_CONFIG_DOC
+```
+
+##### Run OpenSDS and CSI-Plugin Helm Charts
+
+ * Tiller  Permission:
+    Tiller is the in-cluster server component of Helm. By default, helm init installs the Tiller pod into the kube-system namespace, and configures Tiller to use the default service account.
+    
+```
+kubectl create clusterrolebinding tiller-cluster-admin \
+--clusterrole=cluster-admin \
+--serviceaccount=kube-system:default
+```
+
+```
+kubectl create ns opensds
+cd $HOME/kubernetes/opensds-installer/charts
+helm install opensds/ --name={ service_name } --namespace=opensds
+```
+check the result using `kubectl get po -n opensds`
+```$xslt
+NAME                                                   READY   STATUS    RESTARTS   AGE
+opensds-service-opensds-apiserver-54f9dc9776-6zwhv     1/1     Running   0          1h40m
+opensds-service-opensds-authchecker-68c487c885-9zcf6   1/1     Running   0          1h40m
+opensds-service-opensds-controller-9d7b89c7c-4qln8     1/1     Running   0          1h40m
+opensds-service-opensds-dashboard-5f6c5d958b-ddv7d     1/1     Running   0          1h40m
+opensds-service-opensds-db-6cfcd45598-jxx2f            1/1     Running   0          1h40m
+opensds-service-opensds-dock-76b95bf9dd-78smj          1/1     Running   0          1h40m
+
+```
+
+Now you are ready to run csiplugin helm chart
+
+```
+#Please update csiplugin/values.yaml before running csiplugin helm chart
+helm install csiplugin/ --name={ service_name }
+
+```
+
+## Testing steps
+### OpenSDS CLI tool
+#### Download cli tool.
+```
+wget https://github.com/opensds/opensds/releases/download/v0.5.2/opensds-hotpot-v0.5.2-linux-amd64.tar.gz 
```

Comment:
Suggest using the latest version.

Reply:
@leonwanghui Updated latest version, Thanks